test/Li.jl

```diff
@@ -93,6 +93,12 @@ end
         @test PolyLog.li(n, 1//1 + 0//1im) ≈ zeta
         @test PolyLog.li(n, 1 + 0im) ≈ zeta
         @test PolyLog.li(n, BigFloat("1.0") + 0im) == PolyLog.zeta(n, BigFloat)
+
+        #ForwardDiff Test
+        if isdefined(Base,:get_extension)
```

Comment:
Could you please check if all these `if isdefined(...)` statements are still necessary after your recent commits? If not, could you please remove them?